File: Occurrences_After_Bigram.java
class Solution {
	public String[] findOcurrences(String text, String first, String second) {


		String[] temp = text.split(" ");

		List<String> ans = new ArrayList<>();

		for(int i = 2 ; i < temp.length ; i++){

			if(temp[i-2].equals(first)  && temp[i-1].equals(second)){

				ans.add(temp[i]);
			}
		}

		return ans.toArray(new String[0]);
	}
}

File: Occurrences_After_Bigram.cpp
#include<iostream>
class Solution {
public:
    vector<string> findOcurrences(string text, string first, string second) {
        vector <string> s,s1;
        text=text+" ";
        string h;
        int d=0;
        for(int i=0;i<text.length();i++){
           if(text[i]==32){
            
            h= text.substr(d,(i-d));
               d=i+1;
          
           s1.push_back(h);
           }
          
           }
            for(int j=0;j<s1.size()-2;j++)
       {
           if(s1[j]==first && s1[j+1]==second)
               s.push_back(s1[j+2]);
       }
        
        return s;
    }
};
